apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -68,7 +68,9 @@
   margin-right: 4px;
 }
 
-.button-kebab {
+// This is duplicated to ensure it takes specificity over the default
+// button styles. Specifically so that the display is none by default.
+.button-kebab.button-kebab {
```

Comment:
On interesting! Didn't know this was a way to add more specificity. Is this safer than `!important`?

Reply:
Yeah I think so. I only learned about this on Friday!